Library/ViewModels/ReportProjectFormViewModel.swift

```diff
@@ -5,117 +5,108 @@ import ReactiveSwift
 
 public protocol ReportProjectFormViewModelInputs {
   func viewDidLoad()
-
-  /// Submits a report using the createFlagging mutation
-  func submitReport(
-    contentId: String,
-    kind: GraphAPI.FlaggingKind,
-    details: String,
-    clientMutationId: String?
-  )
 }
 
-public protocol ReportProjectFormViewModelOutputs {
-  /// Emits the currently logged in user's email
-  var userEmail: Signal<String, Never> { get }
-}
+public protocol ReportProjectFormViewModelOutputs {}
```

Comment:
So here is an interesting problem: the outputs of our view model are `@Published` properties, and Swift won't let you define a wrapped property like `@Published` in a protocol. That means we can't actually put `userEmail` up here in the outputs, nor any of our other computed properties.

We could potentially define it like this:
```
public protocol ReportProjectFormViewModelOutputs {
   public var retrievedEmailPublisher: Publisher<String?> { get }
   public var retrievedEmail: String?
}
```

but that seems a bit messy. An alternative could also be just making the inputs and outputs a concrete object, instead of a protocol. Or getting rid of them entirely! Thoughts?

Reply:
I could be wrong, but I don't think this input/output protocol pattern is common when using Combine. It seems like more of an RxSwift pattern. I'm happy to explore removing it entirely.  I'm also happy to explore keeping our `@Published` properties more organized in a separate concrete object as you mentioned.

Reply:
Ah, interesting, so maybe there's no good reason to keep it. I'm open to either. It feels a little verbose to me, but it is nice in terms of auto-complete.

Reply:
I vote we get rid of the inputs and outputs. I think with swiftui/combine bindings (where UI and view model stay in sync almost for us) it's not that useful to distinguish between the inputs and the outputs anymore.

Reply:
Fair point!